src/formio/templates/editGrid.ejs

```diff
@@ -1,42 +1,60 @@
-<div class="{{ctx.ofPrefix}}editgrid">
-  <div class="{{ctx.ofPrefix}}editgrid__groups">
-    <ul>
-      {% ctx.rows.forEach(function(row, rowIndex) { %}
-      <div class="{{ctx.ofPrefix}}editgrid__group">
-				{% if (!!ctx.component.groupLabel) { %}<div class="{{ctx.ofPrefix}}editgrid__group-label">{{ ctx.t(ctx.component.groupLabel, { _userInput: true }) }} {{ rowIndex + 1 }}</div>{% } %}
-        <li class="{{ctx.ofPrefix}}editgrid__list" ref="{{ctx.ref.row}}">
-          {{row}}
+<div class="openforms-editgrid">
+  <div>
+    {% ctx.rows.forEach(function(row, rowIndex) { %}
+      <div class="utrecht-form-fieldset openforms-editgrid__item" ref="{{ctx.ref.row}}">
+        <fieldset class="utrecht-form-fieldset__fieldset utrecht-form-fieldset--html-fieldset">
+          {% if (!!ctx.component.groupLabel) { %}
+            <legend class="utrecht-form-fieldset__legend utrecht-form-fieldset__legend--html-legend openforms-editgrid__item-heading">
+              {{ ctx.t(ctx.component.groupLabel, { _userInput: true }) }} {{ rowIndex + 1 }}
+            </legend>
+          {% } %}
+
+          <div>{{row}}</div>
 
           {% if (ctx.openRows[rowIndex] && !ctx.readOnly) { %}
-          <div class="{{ctx.ofPrefix}}editgrid__group-actions {{ctx.ofPrefix}}toolbar {{ctx.ofPrefix}}toolbar--reverse">
-              <ul class="{{ctx.ofPrefix}}toolbar__list">
-              <li class="{{ctx.ofPrefix}}toolbar__list-item">
-                <button class="{{ctx.ofPrefix}}button {{ctx.ofPrefix}}button--primary utrecht-button utrecht-button--html-button utrecht-button--primary-action" ref="{{ctx.ref.saveRow}}">{{ctx.t(ctx.component.saveRow || 'Save', { _userInput: true })}}</button>
-              </li>
+            <p
+              class="utrecht-button-group utrecht-button-group--openforms-editgrid"
+              {% if (ctx.component.removeRow) { %}role="group"{% } %}
+            >
+
+              <button
+                class="openforms-button openforms-button--primary utrecht-button utrecht-button--html-button utrecht-button--primary-action"
+                ref="{{ctx.ref.saveRow}}"
+              >
```

Comment:
Since the latest design tokens are included, the openforms classes are no longer needed I think
```suggestion
              <button
                class="utrecht-button utrecht-button--primary-action"
                ref="{{ctx.ref.saveRow}}"
              >
```

Reply:
oooh I will try that out!